.github/workflows/turborepo.yml

```diff
@@ -24,7 +24,7 @@ jobs:
         run: pnpm audit --prod --audit-level moderate
 
   build:
-    name: Lint, Build, and Test
+    name: Build, Test, and Lint
```

Comment:
currently linting seems to work fine without building dependencies first, but this doesn't seem like a safe assumption going forward.

Since some packages depend on other packages in the repo, there may be generated types/build artifacts that need to be resolved while linting in the future, so I a moved the lint step to happen after building.